recipe/meta.yaml

```diff
@@ -58,9 +54,11 @@ requirements:
     - psutil
     - py-cordex
     - pybtex
-    - pytest-json-report  # needed by esmpy=8.4.0/1 (to be removed in 2.9.0)
+    - python >=3.9
+    - python-stratify >=0.3
     - pyyaml
     - requests
+    - scipy >=1.6
     - shapely
```

Comment:
we have `shapely >=2.0.0` in upstream - shall we do it here too? Also a quick mention on the iris pin differing from upstream would be nice

Reply:
> we have shapely >=2.0.0 in upstream - shall we do it here too? 

That is not included in v2.9, I left it for the next release because I was a bit concerned that there would be very little freedom left in solving the environment.

> Also a quick mention on the iris pin differing from upstream would be nice

The iris pin here matches what is in setup.py